src/DataFramesMeta.jl

```diff
@@ -413,6 +449,11 @@ function orderby(x::GroupedDataFrame, @nospecialize(args...))
     throw(ArgumentError("@orderby with a GroupedDataFrame is reserved"))
 end
 
+function orderby(x::SubDataFrame, @nospecialize(args...))
+    t = DataFrames.select(x, args...)
+    x[sortperm(t), :]
+end
```

Comment:
Why is this in this PR?

Reply:
Because its a bug that Bogumil noticed in the implementation of `@where` that we hadn't caught in the implementation of `@orderby`. I can delete it if you want to keep things separate.